BlackJackGame/Game.cs

```diff
@@ -83,6 +90,38 @@ public List<string> PlayersTurn(List<string> playersHand, Deck deck, List<string
             return playersHand;
         }
         
-        
+
+        public int CalculateScore(List<string> playersHand)
```

Comment:
Prob need to add in our new and improved method of calculating the score right ?